### DISCUSSION TITLE
Scraper with no (official) api

### DISCUSSION THREAD

**Discussion by Skycrox** [jellyfin/jellyfin#12557] [General]
Currently I implement a scraper for fernsehserien.de. Initially for private use only. Maybe, after completion I decide to make it public. 

My question.
Is it okay for the jellyfin developer, that scrapers are published, that uses services without official API?

**Comment by Magnetilo22** [comment]
You can find an answer to your question here: https://forum.jellyfin.org/t-metadata-from-information-sites-without-api

However I can see no problem for jellyfin if you scrape and write to .nfo files.
